src/polyswarmtransaction/transaction.py

```diff
@@ -148,3 +148,26 @@ def __import_transaction_class(module: ModuleType, class_name: str) -> Type[Tran
             raise exceptions.UnsupportedTransactionError(f'{module.__name__}:{class_name} is not a Transaction')
 
         return transaction
+
+
+@dataclasses.dataclass
```

Comment:
I think this needs `(init=False)` 

I forget the exact syntax

Reply:
Looked up the docs on https://docs.python.org/3/library/dataclasses.html#dataclasses.dataclass:
> 

    - init: If true (the default), a __init__() method will be generated.

    If the class already defines __init__(), this parameter is ignored.

The parameter will be ignored if provided, as `__init__` is defined on this class.

Reply:
Perfect